### DISCUSSION TITLE
Best settings for making a video game overworld model

### DISCUSSION THREAD

**Discussion by Daviljoe193** [TheLastBen/fast-stable-diffusion#1626] [Q&A]
I've tried for a good while now to get this right, but I just can't, with the output either being too different from the dataset, or kinda blocky and weird, almost like something from VQGAN+CLIP. The best I have to show is [this model](https://huggingface.co/Kagerage/paper-mario-thousand-year-door-map-model), which is far from optimal, as the results on the page were **_very_** cherry picked, and I could barely get it to produce the same outputs the day after, so it was a dead-end model. I don't remember what settings I used to train it, besides it being trained back on January 10th (Latest everything as of that date, since I always replace the drive cell with !mkdir -p /content/gdrive/MyDrive), and the dataset being made up of 20 overworld images (Sharing a theme of no NPCs, a completely stock camera angle, and the only somewhat reoccuring element being a save block in 7 out of the 20 images), trained on top of the 768 2.1 model. This output (That I made after creating the HF page) captures the exact kind of aesthetic I was going for, that being something that could be mistaken for unused/cut content from a certain game, in this case, Paper Mario: The Thousand Year Door.

![00083-2198025604-ppmriottyd screenshot florida beachside49c2e7b15c7e6bba108c82d975f6e5dbdc2ba04c](https://user-images.githubusercontent.com/67191631/221104140-fa44f147-eb72-4431-a916-5d9531e0d305.png)

Tl;dr, some time over a month ago, I lost days of sleep attempting (and failing) to nail settings for a video game overworld model, and I want some good baseline settings and guidance for a second attempt at this.

Additional info I just remembered! I remember trying just 10/12 images prior, and while I don't have the results on-hand anymore, I remember them looking really bad. At the cost of training time, I'd rather use more images, since this likely will give the model both more thematic consistency, and flexibility.

**Comment by TheLastBen** [comment]
send me the dataset, when I have time I'll try to find the right settings for this type of style

**Reply by Daviljoe193** [reply]
Here it is!
[papermar-768.zip](https://github.com/TheLastBen/fast-stable-diffusion/files/10823263/papermar-768.zip)

**Comment by TheLastBen** [comment]
Do you have a discord ? I'll send the lora to you there

**Reply by Daviljoe193** [reply]
Well... I had a Discord account... the less said about it, the better.

![sorrynodiscord](https://user-images.githubusercontent.com/67191631/221379621-8bacc423-a409-45ac-9579-267531ebcfbc.png)

Google Drive or some other file host will have to work. :\

**Reply by TheLastBen** [reply]
You can start by using simply "daiton" as the prompt
https://drive.google.com/file/d/1YTOXiA31OJNTzYv1x3vVYI_9b8FnLkmr/view?usp=sharing

the model works with v2-512, it can also work with v2-768 but not as good, use the extension https://github.com/kohya-ss/sd-webui-additional-networks
